src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -468,6 +469,7 @@ protected function getFormActions()
                     ->setIcon('trash-bin')
                     ->addExtraClass('btn-outline-danger btn-hide-outline action--delete'));
             }
+            $actions->push(UnsavedChangesIndicator::create('UnsavedChangesIndicator'));
```

Comment:
Is there an appropriate extension we can use instead?  I'd prefer not to put `UnsavedChangesIndicator` in framework if possible, since it can't be used without `silverstripe/admin`.
(GridField shouldn't be in framework either but that's a major version discussion)